staging/src/k8s.io/kubelet/pkg/cri/streaming/server_test.go

```diff
@@ -457,13 +457,15 @@ func doClientStreams(t *testing.T, prefix string, stdin io.Writer, stdout, stder
 func readExpected(t *testing.T, streamName string, r io.Reader, expected string) {
 	result := make([]byte, len(expected))
 	_, err := io.ReadAtLeast(r, result, len(expected))
+	//nolint:testifylint  // always called in go func
```

Comment:
This doesn't seem to stop testifylint from complaining:
```
$ testifylint ./staging/src/k8s.io/kubelet/...
/home/ed/go/src/k8s.io/kubernetes/staging/src/k8s.io/kubelet/pkg/cri/streaming/server_test.go:461:2: require-error: for error assertions use require
/home/ed/go/src/k8s.io/kubernetes/staging/src/k8s.io/kubelet/pkg/cri/streaming/server_test.go:469:2: require-error: for error assertions use require
```

Reply:
nvm, figured this out. It's a golangci-lint instruction and golangci-lint seems to be happy:
```
$ ./hack/verify-golangci-lint.sh -n -a ./staging/src/k8s.io/kubelet/...
installing golangci-lint and logcheck plugin from hack/tools into config=/home/ed/go/src/k8s.io/kubernetes/hack/golangci-hints.yaml ./staging/src/k8s.io/kubelet/...
Congratulations! All files are passing lint :-)
```